dapp/src/pages/DashboardPage/AcreTVLProgress.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import {
+  StackProps,
+  useMultiStyleConfig,
+  HStack,
+  VStack,
+  Grid,
+  Box,
+} from "@chakra-ui/react"
+import { useTVL } from "#/hooks"
+import { BitcoinIcon } from "#/assets/icons"
+import { CurrencyBalance } from "#/components/shared/CurrencyBalance"
+import ProgressBar from "#/components/shared/ProgressBar"
+import { TextMd, TextXs } from "#/components/shared/Typography"
+import { TOTAL_VALUE_LOCKED_CAP } from "#/constants"
+
+type AcreTVLProgressProps = StackProps
+
+const STEP_COUNT = 5
+
+const getSteps = () =>
```

Comment:
Can't it just be a const value?

```suggestion
const getSteps =
```

Reply:
Ref commit: e985c20c